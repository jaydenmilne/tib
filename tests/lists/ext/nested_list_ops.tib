#!desc=Makes sure that operations with nested lists work as expected

{{1,2},{3,4,5}} + {{6,7},{8,9,0}}
{{1,2},{3,4,5}} - {{6,7},{8,9,0}}
{{1,2},{3,4,5}} / {{6,7},{8,9,0}}
{{1,2},{3,4,5}} * {{6,7},{8,9,0}}
{{1,2},{3,4,5}} ^ {{6,7},{8,9,0}}
{{1,2},{3,4,5}} or {{6,7},{8,9,0}}
{{1,2},{3,4,5}} < {{6,7},{8,9,0}}