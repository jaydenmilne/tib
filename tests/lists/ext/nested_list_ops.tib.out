{{7 9} {11 13 5}}
{{-5 -5} {-5 -5 5}}
{{0.166667 0.285714} {0.375 0.444444 inf}}
{{6 14} {24 36 0}}
{{1 128} {6561 262144 1}}
{{1 1} {1 1 1}}
{{1 1} {1 1 0}}
